app/controllers/admin/enterprises_controller.rb

```diff
@@ -8,12 +8,13 @@ module Admin
   class EnterprisesController < Admin::ResourceController
     include GeocodeEnterpriseAddress
     include CablecarResponses
+    include Pagy::Backend
 
     # These need to run before #load_resource so that @object is initialised with sanitised values
     prepend_before_action :override_owner, only: :create
     prepend_before_action :override_sells, only: :create
 
-    before_action :load_enterprise_set, only: :index
+    before_action :load_enterprise_set_on_index, only: :index
```

Comment:
When something is happening only in one action, I often prefer to call it within that action. It keeps related code together and you can see quicker what's happening.

Reply:
I agree 👍 
84e2fbc60a